sycl/include/CL/sycl/builtins.hpp

```diff
@@ -1545,6 +1549,23 @@ detail::enable_if_t<detail::is_genfloatf<T>::value, T> tan(T x) __NOEXC {
 }
 
 } // namespace half_precision
+
+namespace intel {
```

Comment:
I think this should be moved to intel directory.

Reply:
Done